### PR TITLE
support create trigger

### PR DESCRIPTION
**Pull request by zidaye**
refer to ：https://www.postgresql.org/docs/current/sql-createtrigger.html

### CODE REVIEW

File: src/parser.rs [hunk 1]
```diff
@@ -2302,6 +2323,166 @@ impl<'a> Parser<'a> {
...
+                        if lparen_count == 0 {
+                            break;
+                        }
+                        condition_str.push_str(next_token.token.to_string().as_str());
+                    }
+                }
+                Some(condition_str)
+            } else {
+                None
+            };
```
Comment (alamb) [inline src/parser.rs:2384]: I think this code would become much simpler when `condition` is an `Expr`

Something like `let condition = self.parse_expr()`

**Review by alamb — COMMENTED** [review]
Thank you @zidaye  -- this PR appears to have some conflicts. Can you please resolve them?

I also think `condition` should be an Expr

**Comment by alamb** [comment]
This looks very good @zidaye  -- thank you. Please mark this as ready for review when it is ready for another look

**Comment by alamb** [comment]
Closing this PR as it is more than 6 months old without activity. Please reopen or make another PR if you plan to keep working on this.

**Comment by LucaCappelletti94** [comment]
Hi - I need this feature, what is the status on this?

**Comment by alamb** [comment]
I don't think there is any update to report

**Pull request closed without merging**